src/components/Header/Header.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useContext, useState } from 'react';
+import { DispatchContext, StateContext } from '../../context/ToDoContext';
+import classNames from 'classnames';
+
+export const Header: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const { todos } = useContext(StateContext);
+  const dispatch = useContext(DispatchContext);
+
+  const handleCreateNew = () => {
```

Comment:
Consider implementing a check to avoid adding an empty todo 
<img width="673" alt="Screenshot 2024-04-21 at 16 34 36" src="https://github.com/mate-academy/react_todo-app/assets/135232001/ff7f9de2-33f3-44cd-8f85-c35ae714f691">
Also, the title should be trimmed before adding